models/stdp_connection.h

```diff
@@ -185,7 +187,7 @@ class STDPConnection : public Connection< targetidentifierT >
   {
     double_t norm_w =
       ( w / Wmax_ ) - ( alpha_ * lambda_ * std::pow( w / Wmax_, mu_minus_ ) * kminus );
-    return norm_w > 0.0 ? norm_w * Wmax_ : 0.0;
+    return norm_w > 0.0 ? norm_w * Wmax_ : copysign( 0.0, Wmax_ );
```

Comment:
`copysign( 0.0, Wmax_ )` returns a value with the magnitude of its first and the sign of its second argument. So here we would get +0 og -0. Does that make sense?